Yoast/wordpress-seo
# Move RSS tab from Advanced to Titles & Meta's

## Summary

This PR can be summarized in the following changelog entry:

* Moved the RSS tab from Advanced to Titles & Meta's.

## Relevant technical choices:

* Deletes the `wpseo_rss` option.

## Test instructions

This PR can be tested by following these steps:

* Go to Titles & Meta's, see the RSS tab, make sure all the functionality around it still works.

## Quality assurance


Relates to #8796
